Documentation/components/k8s/developers.rst

```diff
@@ -7,37 +7,7 @@ Creating the Docker Images
 Docker 18.06 is needed to build the images, and the daemon has to be started with the `--experimental` flag.
 `See this issue to have more information <https://github.com/moby/moby/issues/32507>`_.
 
-You also need to ``export DOCKER_BUILDKIT=1`` before building the image
-
-::
-
-    docker build --build-arg KUBERNETES=true -t polycubenetwork/k8s-pod-network -f Dockerfile .
-    docker push polycubenetwork/k8s-pod-network
```

Comment:
Not clear to me:
1) why this has to be deleted
2) the meaning of the "export DOCKER_BUILDKIT...". When there are commands such that, I would suggest to add a brief sentence telling people what those are about.

Reply:
No sure what you mean by deleted.
That comment is already present, please look at the full commit.